v1/acl.go

```diff
@@ -20,7 +20,7 @@ func (a *ACL) ACLApi() *client.ACLApiService {
 }
 
 // returns arraySchema
-func (a *ACL) GetACLPolicies(ctx context.Context) (*[]client.ACLPolicyListStub, *QueryMeta, error) {
+func (a *ACL) ListPolicies(ctx context.Context) (*[]client.ACLPolicyListStub, *QueryMeta, error) {
```

Comment:
```suggestion
func (a *ACL) Policies(ctx context.Context) (*[]client.ACLPolicyListStub, *QueryMeta, error) {
```

I thought the name change we talked about here was to just name this `Policies`. 

Did I remember that wrong?

Reply:
Nah -- I likely misheard  🤷 
I'll make the changes!